querybook/webapp/components/BoardRightSidebar/BoardRightSidebar.tsx

```diff
@@ -43,16 +45,18 @@ export const BoardRightSidebar: React.FunctionComponent<IProps> = ({
                 onClick={onCollapse}
                 disabled={isEditMode}
             />
-            <IconButton
-                icon={isEditMode ? 'Check' : 'Edit2'}
-                tooltip={
-                    isEditMode
-                        ? 'Done Editing'
-                        : 'Edit Mode: reorder & delete items'
-                }
-                tooltipPos="left"
-                onClick={onEditModeToggle}
-            />
+            {isEditable && (
+                <IconButton
```

Comment:
if it is not edible, disable this button instead of hiding it (so that the users would know where to look)?